### ISSUE TITLE
Don't read from stderr when running commands

### ISSUE THREAD

**Issue by MarTango** [Silex/docker.el#119]
If you run `docker-compose-exec`, the command `docker-compose config --services` lists unset environment variables to stderr, which then gets read in as docker-compose services.

I think the stderr output should still be piped to *Messages*, but not parsed. (This will involve refactoring to use another function instead of `shell-command-to-string`.

**Comment by Silex** [comment]
@MarTango: can you clarify what you mean by "unset environment variables to stderr"?

Here is what happend when I used the exec popup to set an environment variable:

```
docker-compose  config --services
docker-compose  exec -e FOO=BAR test1 test
```

The environment variable set in the dialog persisted...

**Comment by MarTango** [comment]
Sorry for the ambiguity, I mean when a `docker-compose.yml` file contains [environment variables ](https://docs.docker.com/compose/compose-file/#variable-substitution) that are not set.

e.g.


```
martango $ cat docker-compose.yml
version: "3"
services:
  my_service:
    image: "ubuntu:latest"
    environment:
      - HELLO=${HELLO}

martango $ docker-compose config --services
WARNING: The HELLO variable is not set. Defaulting to a blank string.
my_service
```

**Comment by Silex** [comment]
Ah right, okay fixed by fe74a499ce3246fb9a7d72e6931864b94ce5261d
